inception-ui-kb/src/main/java/de/tudarmstadt/ukp/inception/ui/kb/ConceptTreePanel.java

```diff
@@ -127,13 +127,34 @@ protected boolean isSelected()
         addLink.add(new Label("label", new ResourceModel("concept.list.add")));
         addLink.add(new WriteProtectionBehavior(kbModel));
         add(addLink);
+        /**
```

Comment:
This change is not related to this refactoring. Please create a new pull request for feature changes.